source/features/clean-conversation-headers.tsx

```diff
@@ -1,68 +1,87 @@
 import './clean-conversation-headers.css';
+import React from 'dom-chef';
 import select from 'select-dom';
-import {observe} from 'selector-observer';
+import elementReady from 'element-ready';
+import {ArrowLeftIcon} from '@primer/octicons-react';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import getDefaultBranch from '../github-helpers/get-default-branch';
+import onConversationHeaderUpdate from '../github-events/on-conversation-header-update';
 
-const deinit: VoidFunction[] = [];
+async function initIssue(): Promise<void> {
+	const byline = await elementReady('.gh-header-meta .flex-auto:not(.rgh-clean-conversation-header)');
+	if (!byline) {
+		return;
+	}
 
-function initIssue(): void {
-	const observer = observe('.gh-header-meta .flex-auto:not(.rgh-clean-conversation-header)', {
-		add(byline) {
-			byline.classList.add('rgh-clean-conversation-header');
-			const {childNodes: bylineNodes} = byline;
-			// Removes: octocat opened this issue on 1 Jan [·] 1 comments
-			bylineNodes[4].textContent = bylineNodes[4].textContent!.replace('·', '');
+	byline.classList.add('rgh-clean-conversation-header');
 
-			// Removes: [octocat opened this issue on 1 Jan] · 1 comments
-			for (const node of [...bylineNodes].slice(0, 4)) {
-				node.remove();
-			}
-		}
-	});
-	deinit.push(observer.abort);
+	const {childNodes: bylineNodes} = byline;
+	// Removes: [octocat opened this issue on 1 Jan] · 1 comments
+	for (let i = 0; i < 4; i++) {
+		bylineNodes[0].remove();
```

Comment:
This was the code I suggested
```suggestion
	// Removes: [octocat opened this issue on 1 Jan] · 1 comments
	for (let i = 0; i < 4; i++) {
		byline.childNodes[0].remove();
```